sycl/source/detail/queue_impl.cpp

```diff
@@ -57,7 +59,9 @@ event queue_impl::memcpy(shared_ptr_class<detail::queue_impl> Impl, void *Dest,
   if (Context.is_host())
     return event();
```

Comment:
Does this also need to "add" the event?  Or is the host device just being really simple about these things?

Reply:
Since this branch returns a "ready" event, there's no need to wait for it.